contracts/test-utils/src/constants.ts

```diff
@@ -84,4 +84,9 @@ export const constants = {
     PPM_DENOMINATOR: 1e6,
     PPM_100_PERCENT: 1e6,
     MAX_CODE_SIZE: 24576,
+    ERC20_PROXY_ID: '0xf47261b0',
```

Comment:
I believe we have all of these in the `@0x/types` package (`AssetProxyId` type).